File: domain/src/main/java/de/heilsen/ganzhornfest/Club.java
package de.heilsen.ganzhornfest;


public class Club implements DisplayableItem {
    private String name;
    private String description;

    public Club(String name) {
        this(name, "");
    }

    public Club(String name, String description) {
        this.name = name;
        this.description = description;
    }

    @Override
    public String toString() {
        return "Club{" +
               "name='" + name + '\'' +
               ", description='" + description + '\'' +
               '}';
    }

    public String getName() {
        return name;
    }

    public String getDescription() {
        return description;
    }
}

File: domain/src/main/java/de/heilsen/ganzhornfest/domain/entity/Club.kt
package de.heilsen.ganzhornfest.domain.entity

import java.util.Collections.emptyList

data class Club @JvmOverloads constructor(val name: String = "",
                                          val description: String = "",
                                          val foodList: List<Food> = emptyList<Food>(),
                                          val drinkList: List<Drink> = emptyList<Drink>(),
                                          val actionableOfferList: List<ActionableOffer> = emptyList<ActionableOffer>(),
                                          val poi: Poi = Poi(0.0, 0.0))
